guava/src/main/scala/magnolify/guava/semiauto/FunnelDerivation.scala

```diff
@@ -59,10 +61,10 @@ trait FunnelImplicits {
   implicit val intFunnel: Funnel[Int] = Funnels.integerFunnel().asInstanceOf[Funnel[Int]]
   implicit val longFunnel: Funnel[Long] = Funnels.longFunnel().asInstanceOf[Funnel[Long]]
   implicit val bytesFunnel: Funnel[Array[Byte]] = Funnels.byteArrayFunnel()
-  implicit val booleanFunnel: Funnel[Boolean] = funnel[Boolean](_.putBoolean(_))
-  implicit val byteFunnel: Funnel[Byte] = funnel[Byte](_.putByte(_))
-  implicit val charFunnel: Funnel[Char] = funnel[Char](_.putChar(_))
-  implicit val shortFunnel: Funnel[Short] = funnel[Short](_.putShort(_))
+  implicit val booleanFunnel: Funnel[Boolean] = funnel[Boolean](_.putBoolean(_): @nowarn)
```

Comment:
Are the warnings about boxing or something? Could explicit call the associated box / unbox methods  on these - e.g. `Boolean.box` / `Boolean.unbox`